cypress/e2e/example.cy.js

```diff
@@ -16,4 +16,11 @@ describe("the homepage", () => {
     cy.get('a[id="liedle-btn"]').click()
     cy.url().should('eq', 'http://localhost:1234/liedle/index.html')
   });
+
+  it("2048 Expert link redirects to its web page", () => {

```

Comment:
This test fails. Run `npm test` to run the Cypress tests. You will need to add a link to your webpage in `src/index.html`. 
![image](https://user-images.githubusercontent.com/78716153/183580715-bf0ab503-7b27-4d6c-ae4a-cbd75e79186a.png)